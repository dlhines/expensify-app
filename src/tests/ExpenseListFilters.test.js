import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListFilters from '../components/ExpenseListFilters';
import { filters, altFilters } from './fixtures/filters';

let setTextFilter, sortByDate, sortByAmount, setStartDate, setEndDate, wrapper;

beforeEach(() => {
    setTextFilter = jest.fn();
    sortByDate = jest.fn();
    sortByAmount = jest.fn();
    setStartDate = jest.fn();
    setEndDate = jest.fn();
    wrapper = shallow(<ExpenseListFilters 
        filters={filters} 
        setTextFilter={setTextFilter}
        sortByAmount={sortByAmount}
        sortByDate={sortByDate}
        setStartDate={setStartDate}
        setEndDate={setEndDate}
    />);
});

test('should take ExpenseListFilters snapshot', () => {
    expect(wrapper).toMatchSnapshot();
})
