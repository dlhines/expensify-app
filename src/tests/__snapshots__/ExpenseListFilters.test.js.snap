// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take ExpenseListFilters snapshot 1`] = `undefined`;
